### DISCUSSION TITLE
Unable to clone

### DISCUSSION THREAD

**Discussion by ranu1993** [community/community#65783] [Apps, API and Webhooks] [answered]
### Select Topic Area

General

### Body

Team,

I could not clone JWT token tool even after use my personal token, Can you help me
![kai-2023-09-03-00-01-21](https://github.com/community/community/assets/84901738/a4cf126a-6725-43c9-bb8d-c0a2d3ae6565)

**Comment by jesse-sealand** [comment]
How long ago did you create your Personal Access Token?

**Comment by ranu1993** [comment, marked as answer]
Dear Jesse,

Now it is working thanks for your help

Regards,
ranujan
